packages/arb-token-bridge-ui/src/components/BuyPanel/BackButton.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import { ChevronLeftIcon } from '@heroicons/react/24/outline';
+import Link from 'next/link';
+import { useSearchParams } from 'next/navigation';
+
+import { PathnameEnum } from '@/bridge/constants';
+import { useMode } from '@/bridge/hooks/useMode';
+
+import { Button } from '../common/Button';
+
+export function BackButton() {
+  const { embedMode } = useMode();
+  const searchParams = useSearchParams();
+
+  return (
+    <Link
+      href={{
+        pathname: embedMode ? PathnameEnum.EMBED_BUY : PathnameEnum.BUY,
+        query: searchParams.toString(),
+      }}
+      className="flex flex-row justify-content items-center absolute top-4 left-4 gap-2 hover:opacity-80"
+    >
+      <Button
```

Comment:
Button inside Anchor element is invalid: https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

Reply:
Maybe we can have `<Link as={Button...}>`?